bdai_ros2_wrappers/bdai_ros2_wrappers/time.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Boston Dynamics AI Institute Inc.  All rights reserved.
+
+from datetime import datetime, timedelta
+from typing import Union
+
+from rclpy.duration import Duration
+from rclpy.time import Time
+
+
+def as_proper_time(time: Union[int, float, datetime, Time]) -> Time:
+    """Return `time` as a proper Time object."""
+    if isinstance(time, int):
+        return Time(nanoseconds=time)
+    if isinstance(time, float):
+        return Time(seconds=time)
```

Comment:
I feel that this float/int discrepancy could lead to confusion. I could potentially see someone trying to call this function to represent 5 seconds by passing in 5 instead of 5.0 and instead getting a representation that corresponds to 5 ns. Is there a way we can make this distinction more clear?

Reply:
That is a possibility, yes. The fool proof solution is to stop taking scalars for time/duration and force users to use explicit representations. That makes it somewhat inconvenient when you add `numpy`, `scipy`, `pandas`, etc. to the mix, but you can't go wrong that way. The second best solution is to assume SI units for scalars -- won't save you from inadvertently passing in a nanosecond count.

Reply:
Totally agree with that -- I guess I'm wondering if there a clean way to specify what type corresponds to what unit in the docstring to try to mitigate this?